pkg/gateway/multiparts/multipart.pb.go

```diff
@@ -0,0 +1,218 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
Should we commit this generated file?

Reply:
As in graveler - the pb files are committed to prevent repository from breaking

Reply:
Not all generated code is committed, but I'm fine with keeping Graveler conventions.